quantum/wpm.c

```diff
@@ -54,6 +96,10 @@ void update_wpm(uint16_t keycode) {
         }
         wpm_timer = timer_read();
     }
+
+#ifdef SERIAL_LINK_ENABLE
+    send_current_wpm();
+#endif
```

Comment:
Hmm... After understanding the serial_link system a bit more, I'm not entirely sure about this. Maybe this should be moved up to inside the if statements above (just after `current_wpm = ...`), and copied to the corresponding place in `decay_wpm` below?